doc/user/content/ops/troubleshooting.md

```diff
@@ -10,58 +10,120 @@ aliases:
 ---
 
 You can use the queries below for spot debugging, but it may also be
-helpful to make them more permanent tools by creating them as views for `SUBSCRIBE`ing or materialized views to read more efficiently. Note that the
-existence of these additional views may itself affect performance.
+helpful to monitor their output by `SUBSCRIBE`ing to their change stream, e.g.,
+by issuing a command `COPY (SUBSCRIBE (<query>) TO stdout`.
+Note that this additional monitoring may, however, itself affect performance.
+
+### Limitations on Introspection Sources and Views for Troubleshooting
+
+Importantly, the introspection sources and views used in the queries below
+should be considered subject to changes in the future without further notice.
+This is why these sources and views are all created in the `mz_internal`
+schema, and it is not possible to create higher-level views depending on them.
+
+Introspection sources are maintained by independently collecting internal logging
+information within each of the replicas of a cluster. Thus, in a multi-replica
+cluster, the queries below need to be directed to a specific replica by issuing
+the command `SET cluster_replica = <replica_name>`. Materialize also directly
+exposes replica-specific introspection sources by suffixing the respective catalog
+relation names with a replica number. For example, `mz_internal.mz_compute_frontiers_1`
+corresponds to the introspection source `mz_internal.mz_compute_frontiers` in the
+first replica of the cluster.
+
+As a consequence of the above, you should expect the answers to the queries below
+to vary dependending on which cluster you are working in. In particular, indexes
+and dataflows are local to a cluster, so their introspection information will
+vary across clusters.
 
 ### How fast are my sources loading data?
 
-You can count the number of records accepted in a source or materialized view.
-Note that this makes less sense for a non-materialized view,
-as invoking it will create a new dataflow and run it to the point that
-it is caught up with its sources; that elapsed time may be informative,
-but it tells you something other than how fast a collection is populated.
+You can count the number of records accepted in a source, materialized view,
+or indexed view. Note that this makes less sense for a non-materialized,
+non-indexed view, as invoking it will create a new dataflow and run it to
+the point that it is caught up with its sources; that elapsed time may be
+informative, but it tells you something other than how fast a collection
+is populated.
 
 ```sql
 -- Report the number of records available from the materialization.
-select count(*) from my_source_or_materialized_view;
+SELECT count(*) FROM my_source_or_materialized_or_indexed_view;
```

Comment:
Since `materialization` seems to be a word we subsequently use to mean "source or materialized or indexed view" (or just "materialized or indexed view"?), you could shorten that relation name to `mz_materialization` (or `my_source_or_materialization`?).

Reply:
Yes, my understanding is that our documentation was using the term 'materialization' originally to refer to what we call now an indexed view (i.e., in-memory materialization), but now the term has been encompassing also materialized views (i.e., in-storage materialization). Sources are not covered by the term, as far as I can tell. So I changed to `my_source_or_materialization`.